rest-api/alembic/versions/508708c67d9f_initial_metrics.py

```diff
@@ -0,0 +1,287 @@
+"""Initial metrics
+
+Revision ID: 508708c67d9f
+Revises: 51415576d3e9
+Create Date: 2017-12-08 18:44:08.904095
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import model.utils
+
+
+from participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from participant_enums import WithdrawalStatus, SuspensionStatus
+from participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType
+from participant_enums import MetricSetType, MetricsKey
+from model.site_enums import SiteStatus
+from model.code import CodeType
+
+# revision identifiers, used by Alembic.
+revision = '508708c67d9f'
+down_revision = '51415576d3e9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(u'biobank_order_ibfk_4', 'biobank_order', type_='foreignkey')
```

Comment:
Still trying to figure out what's going on here @danqrodney ; let me know if you have any ideas.

Seems some kind of bad interaction with setting the schema vs the ORM foreign key stuff we're using.